class.photon.php

```diff
@@ -695,6 +700,12 @@ public function filter_srcset_array( $sources, $size_array, $image_src, $image_m
 			$newsources = null;
 
 			foreach ( $multipliers as $multiplier ) {
+
+				/** This filter is already documented in class.photon.php */
+				if ( apply_filters( 'jetpack_photon_skip_image', false, $url ) ) {
```

Comment:
We can move 675/680 up so we set `$url` earlier and still use that. Cleaner than having that twice in there.